README.md

```diff
@@ -1,57 +1,269 @@
-[codecov-url]:https://codecov.io/gh/moxystudio/{package-name}
-[codecov-image]:https://img.shields.io/codecov/c/github/moxystudio/{package-name}/master.svg
-[david-dm-url]:https://david-dm.org/moxystudio/{package-name}
-[david-dm-image]:https://img.shields.io/david/moxystudio/{package-name}.svg
-[david-dm-dev-url]:https://david-dm.org/moxystudio/{package-name}?type=dev
-[david-dm-dev-image]:https://img.shields.io/david/dev/moxystudio/{package-name}.svg
+[npm-url]:https://npmjs.org/package/@moxy/react-accordion
+[downloads-image]:https://img.shields.io/npm/dm/@moxy/react-accordion.svg
+[npm-image]:https://img.shields.io/npm/v/@moxy/react-accordion.svg
+[build-status-url]:https://github.com/moxystudio/react-accordion/actions
+[build-status-image]:https://img.shields.io/github/workflow/status/moxystudio/react-accordion/Node%20CI/master
+[codecov-url]:https://codecov.io/gh/moxystudio/react-accordion
+[codecov-image]:https://img.shields.io/codecov/c/github/moxystudio/react-accordion/master.svg
+[david-dm-url]:https://david-dm.org/moxystudio/react-accordion
+[david-dm-image]:https://img.shields.io/david/moxystudio/react-accordion.svg
+[david-dm-dev-url]:https://david-dm.org/moxystudio/react-accordion?type=dev
+[david-dm-dev-image]:https://img.shields.io/david/dev/moxystudio/react-accordion.svg
 
-{package-drescription}
+A react component that renders an accessible accordion.
 
 ## Installation
 
 ```sh
-$ npm install @moxy/{package-name}
+$ npm install @moxy/react-accordion
 ```
 
 This library is written in modern JavaScript and is published in both CommonJS and ES module transpiled variants. If you target older browsers please make sure to transpile accordingly.
 
 ## Motivation
 
-{package-motivation}
+An accordion allows users to toggle the display of sections of content. This package was created to make easier to implement an accordion with accessible navigation.
 
 ## Usage
 
-{package-usage-example}
+```js
+import React from 'react';
+import { Accordion, AccordionItem } from '@moxy/react-accordion';
+
+const MyComponent = () => {
+    <div>
+        <Accordion defaultActiveItems={ [0, 1] } multiple scrollIntoView>
+            <AccordionItem title="Foo">
+                Bar
+            </AccordionItem>
+            <AccordionItem title="Foo1">
+                Bar1
+            </AccordionItem>
+        </Accordion>
+    </div>
+}
+```
 
 ## API
 
-{package-api-description}
+These are the props available in [`@moxy/react-accordion`](https://github.com/moxystudio/react-accordion).
+
+### Accordion Props
+
+#### children
+
+Type: `node` | Required: `true`
+
+One or more `AccordionItem` elements that will be rendered inside the `Accordion`.
+
+#### icon
+
+Type: `any` | Required: `false`
+
+Content of the icon displayed for each item.
+
+#### defaultActiveItems
+
+Type: `array` | Required: `false` | Default: `[]`
+
+The indices of the items that should be open by default.  
```

Comment:
Super nitpick, but as we use `index` in the context of dev, I'd maintain the consistency.

```suggestion
The indexes of the items that should be open by default.  
```